docs/pages/example/geojson-layer-in-stack.html

```diff
@@ -9,6 +9,13 @@
 
 map.on('load', function () {
 
+    // Find the index of the bottom-most type: symbol layer in the map style
```

Comment:
> // Find the index of the bottom-most type: symbol layer in the map style
👇 
> // Find the index of the bottom-most symbol layer in the map style

(sorry I know this was my wording before but I forgot that you can't do code highlighting in js comments so this looks kind of confusing in plain text)

Reply:
also this comment should be right before line 15 (the line where we're actually finding the symbol layer)